awx/ui/client/src/templates/job_templates/edit-job-template/job-template-edit.controller.js

```diff
@@ -609,7 +609,7 @@ export default
 
                         Rest.setUrl(data.related.labels);
 
-                        var defers = [credDefer];
+                        var defers = [credDefer, instanceGroupDefer];
```

Comment:
This bug already existed, but I started running into it a lot when testing the changes to the IG service. We need to add the IG (dis)association promises to the list of defers we wait for on save before reloading the form. Otherwise, we will sometimes reload the page before all of the association requests have completed. This makes it seem like some of the associations weren't made when they actually were.

Reply:
👍